images/linux/scripts/helpers/github-api.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+################################################################################
+##  File:  github-api.sh
+##  Desc:  Helper functions for github api
+################################################################################
+
+function getLatestReleaseURI {
+     if [[ $# -eq 0 || "x$1" = "x-h" || "x$1" = "x--help" ]];then
```

Comment:
This function is intended to run in CI so we don't need to support help argument for it.
To be consistent, it is better to put some synopsis comment under function

Reply:
In fact this code intended to provide the logs easy to find a reason of fail. But ok. "-h" keys are unneeded in fact.